completions/_vagrant

```diff
@@ -14,6 +14,15 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+	        if [ "$2" = "init" ]; then
```

Comment:
RE: Changing $2 to $1 causes the if to never be true since $1 would be `vagrant`.
RE: further thinking: Gotcha, updated the boxes method.